javascript/ql/lib/semmle/javascript/security/dataflow/UnsafeJQueryPluginQuery.qll

```diff
@@ -23,6 +23,9 @@ class Configuration extends TaintTracking::Configuration {
     node instanceof DomBasedXss::Sanitizer
     or
     node instanceof Sanitizer
+    or
+    // Plugins usually do `$(this)` to coerce an existing DOM element to a jQuery object.
+    node instanceof DataFlow::ThisNode
```

Comment:
Shouldn't this be a new subclass of `UnsafeJQueryPluginCustomizations::Sanitizer`?  

Also, I'm interested in a test of this, to see what the full FP looks like.

Reply:
> Shouldn't this be a new subclass of UnsafeJQueryPluginCustomizations::Sanitizer?

Done 👍 

> Also, I'm interested in a test of this, to see what the full FP looks like.

Also added a test. There's a commit that adds the test without the sanitizer and the next commit updates the output to show the effects of the sanitizer.